src/solver-service/lib/opam_repository_intf.ml

```diff
@@ -1,17 +1,21 @@
 module type S = sig
-  val open_store : unit -> Git_unix.Store.t Lwt.t
 
-  val clone : unit -> unit Lwt.t
-  (** [clone ()] ensures that "./opam-repository" exists. If not, it clones it. *)
+val open_store : ?repo_url:string -> unit -> Git_unix.Store.t Lwt.t
+(** Open the local clone of the repo at the given URL.
+    If the local clone does not yet exist, this clones it first.
+    If repo_url is unspecified, it defaults to ocaml/opam-repository on GitHub. *)
```

Comment:
Keep the path of `"./opam-repository"` in the comment if that is still the directory used.

Reply:
It's slight different, this is not valid in the comment because of `repo_url` is always used.
```
let open_store ?(repo_url = default_repo_url) () =
      clone ~repo_url () >>= fun () -> ...
```